Fix unmanaged-cluster GitHub Actions workflows - Build failure issue

Example pipelines failing due to build failure -

https://github.com/vmware-tanzu/community-edition/actions/runs/2221931724

https://github.com/vmware-tanzu/community-edition/actions/runs/2221934370

https://github.com/vmware-tanzu/community-edition/actions/runs/2228697051

https://github.com/vmware-tanzu/community-edition/actions/runs/2228988319

https://github.com/vmware-tanzu/community-edition/actions/runs/2229114021

The error in all of them looks something like this -

```
+ rm -f hack/release/../../release/tce-linux-amd64-v0.13.0-dev.1/tanzu
+ mv -f hack/release/../../release/tce-linux-amd64-v0.13.0-dev.1/linux-amd64-standalone/tanzu-core-linux_amd64 hack/release/../../release/tce-linux-amd64-v0.13.0-dev.1/tanzu
mv: cannot stat 'hack/release/../../release/tce-linux-amd64-v0.13.0-dev.1/linux-amd64-standalone/tanzu-core-linux_amd64': No such file or directory
Makefile:270: recipe for target 'package-release' failed
make[2]: *** [package-release] Error 1
make[2]: Leaving directory '/home/ubuntu/actions-runner/_work/community-edition/community-edition'
+ error 'TCE BUILD FAILED!'
+ printf '\E[31m'
+ echo 'TCE BUILD FAILED!'
TCE BUILD FAILED!
+ printf '\E[0m'
+ exit 1
2022/04/26 20:20:00 exit status 1
exit status 1
FAIL	github.com/vmware-tanzu/community-edition/cli/cmd/plugin/unmanaged-cluster/test/e2e	425.558s
make[1]: *** [e2e-test] Error 1
Makefile:27: recipe for target 'e2e-test' failed
make[1]: Leaving directory '/home/ubuntu/actions-runner/_work/community-edition/community-edition/cli/cmd/plugin/unmanaged-cluster'
make: *** [unmanaged-cluster-e2e-test] Error 2
Makefile:518: recipe for target 'unmanaged-cluster-e2e-test' failed
```

--- a/Makefile
+++ b/Makefile
@@ -515,6 +515,6 @@ vsphere-management-and-workload-cluster-e2e-test:
 	BUILD_VERSION=$(BUILD_VERSION) test/vsphere/run-tce-vsphere-management-and-workload-cluster.sh
 
 unmanaged-cluster-e2e-test:
-	cd cli/cmd/plugin/unmanaged-cluster && make e2e-test BUILD_VERSION=$(BUILD_VERSION)
+	cd cli/cmd/plugin/unmanaged-cluster && BUILD_VERSION=$(BUILD_VERSION) make e2e-test
 
 ##### E2E TESTS

